build/steps/build-windows.yml

```diff
@@ -40,6 +40,19 @@ jobs:
       inputs:
         versionSpec: ${{ parameters.nugetVersion }}
 
+    - task: UseDotNet@2
+      displayName: 'Install .NET Core 3 preview sdk'
+      inputs:
+        packageType: sdk
+        version: 3.0.100-preview7-012821
```

Comment:
Pin the version or just use the latest?

Reply:
I changed it to use the latest 3.0.x preview version.

Reply:
Used the latest preview in the new build as expected :)
`Downloading: https://download.visualstudio.microsoft.com/download/pr/10927870-88e5-4b7d-9a5b-1a7cb46a98c0/c17f892b315e5f44052c30cd2eb21023/dotnet-sdk-3.0.100-rc1-014190-win-x64.zip`